### PR TITLE
added getAutoGeneratedCommentForPath.ts

### PR DESCRIPTION
**Pull request by sahandfarahani**

### CODE REVIEW

**Comment by sahandfarahani** [comment]
> Where is the tests?

oh should I have push the tests as well?

**Comment by ralph-dev** [comment]
Yes both the Test for the Function + Function in PR

File: src/utils/test.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { expect } from "chai";
+import getAutoGeneratedCommentForPath from "./getAutoGeneratedCommentForPath";
+
+describe("Get getAutoGeneratedCommentForPath Function", () => {
+  it("Should retrieve the auto generated comment for common folder", () => {
```
Comment (ralph-dev) [inline src/utils/test.ts:5]: Need to test worst case of Empty String

File: src/utils/test.ts [hunk 2]
```diff
@@ -7,3 +7,15 @@ describe("getAutoGeneratedCommentForPath Function", () => {
...
+  it("Should return empty string for non-common folder", () => {
+    const result = getAutoGeneratedCommentForPath("sssssssss");
+    expect(result).to.equal("");
+  });
+});
+describe("getAutoGeneratedCommentForPath Function", () => {
+  it("Should return empty string for empty string", () => {
+    const result = getAutoGeneratedCommentForPath("");
+    expect(result).to.equal("");
+  });
```
Comment (ralph-dev) [inline src/utils/test.ts:20]: @sahandfarahani @Manpreet-Bhatti One Describe Per Function. Multiple IT per Function.
